docs/utilities/tags.md

```diff
@@ -9,12 +9,11 @@ Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid
 Similarly, you may append to or override tags declared in other domains, such as Vanilla, by declaring your own JSONs.
 For example, to add your own mod's saplings to the Vanilla sapling tag, you would specify it in `/data/minecraft/tags/blocks/saplings.json`, and Vanilla will merge everything into one tag at reload, if the `replace` option is false.
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
+Entries which may not be present, such as resources from other mods, can be included using a compound tag that uses an id string and required boolean set to false.
```

Comment:
Reading this does not make much sense. Compound tags are not a thing in JSON. There is an alternative representation in JSON objects, but they are not the same. Provide an example when stating a case.

Additionally, when saying that making something optional, you should clearly express that point. (Ex. `If a tag entry should not cause the tag to fail to load...`)

Reply:
Interesting.  I used the term since it was the alt text assigned to the image that labeled the relevant part in the vanilla wiki. I'll do some reading regarding the json standard and see what the proper term is.

Reply:
Yes, data is stored either using the NBT format (to which objects are represented as Compound Tags) or JSON format (to which objects are represented as JSON Objects). Tags can be serialized in both formats, though the one datapacks use is the JSON format.